cli.js

```diff
@@ -3,8 +3,6 @@
 'use strict';
 var multiline = require('multiline');
 var updateNotifier = require('update-notifier');
-var getStdin = require('get-stdin');
-var subarg = require('subarg');
```

Comment:
We need that one for parsing subcommands, like `$ pageres [ foo.com 1024x100 ] [bar.com 800x600]`